coralproject/talk
# Add translations

## What does this PR do?

Adds a few more strings to translations.

## How do I test this PR?

- Change to spanish
- Check translations are there for configure stream and dashboard.
